Sources/Rendering/OpenGL/VolumeMapper/index.js

```diff
@@ -232,18 +232,31 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
     ).result;
 
     const averageIPScalarRange = model.renderable.getAverageIPScalarRange();
+    let min;
+    let max;
 
-    // TODO: Adding the .0 at the end feels hacky
+    // If min or max is not already a float.
+    // make them into floats for glsl
+    if (Number.isInteger(averageIPScalarRange[0])) {
+      min = `${averageIPScalarRange[0]}.0`;
+    } else {
+      min = `${averageIPScalarRange[0]}`;
+    }
+    if (Number.isInteger(averageIPScalarRange[1])) {
+      max = `${averageIPScalarRange[1]}.0`;
+    } else {
+      max = `${averageIPScalarRange[1]}`;
+    }
```

Comment:
Why not use [toFixed](https://caniuse.com/?search=toFixed)() instead?

something like this 

```js
const a = averageIPScalarRange[0];
const b = averageIPScalarRange[1];

min = (Number.isInteger(a)) ? a.toFixed(1).toString() : (a.toString());
max = (Number.isInteger(b)) ? b.toFixed(1).toString() : (b.toString());
```
ex : 
```js
let i = 1
(Number.isInteger(i)) ? i.toFixed(1).toString() : (i.toString());
"1.0"
i = 1.25
(Number.isInteger(i)) ? i.toFixed(1).toString() : (i.toString());
"1.25"
```

Reply:
Oh I like that, looks clean and works well! I changed it to this. Never used toFixed before